Missing Dependency `react-dev-utils` (?)

Just installed latest `0.1.7`.

Windows 10,
node.js 7.0.0
npm 3.10.8

Created the app and tried to run with `elm-app start`.

Got the following exception:

```
D:\Projects\Elm\my-app>elm-app start
module.js:474
    throw err;
    ^

Error: Cannot find module 'react-dev-utils/webpackHotDevClient'
    at Function.Module._resolveFilename (module.js:472:15)
    at Function.resolve (internal/module.js:27:19)
    at Object.<anonymous> (C:\Users\Me\AppData\Roaming\npm\node_modules\create-elm-app\config\webpack.config.dev.js:13:13)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
    at Module.require (module.js:500:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\Users\Me\AppData\Roaming\npm\node_modules\create-elm-app\scripts\start.js:5:16)
    at Module._compile (module.js:573:32)
    at Object.Module._extensions..js (module.js:582:10)
    at Module.load (module.js:490:32)
    at tryModuleLoad (module.js:449:12)
    at Function.Module._load (module.js:441:3)
```

I have checked `package.json` and there is no `react-dev-utils` listed as a dependency.

Thank you.

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "create-elm-app",
-  "version": "0.1.7",
+  "version": "0.1.8",
   "description": "Elm apps with zero configuration",
   "keywords": [
     "elm",
@@ -26,7 +26,7 @@
   },
   "bugs": {
     "url": "https://github.com/halfzebra/create-elm-app/issues",
-    "email": "duardeco@gmail.com"
+    "email": "eduard.kyvenko@gmail.com"
   },
   "files": [
     "bin",
@@ -54,10 +54,10 @@
     "html-webpack-plugin": "^2.24.1",
     "install": "0.8.1",
     "minimist": "^1.2.0",
-    "opn": "^4.0.2",
     "path-exists": "^3.0.0",
     "postcss-loader": "^1.1.0",
     "prompt": "1.0.0",
+    "react-dev-utils": "^0.3.0",
     "style-loader": "0.13.1",
     "webpack": "^1.13.3",
     "webpack-dev-server": "1.16.2"

